.github/workflows/black.yml

```diff
@@ -0,0 +1,15 @@
+name: Black lint
+
+on: [push, pull_request]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: psf/black@stable
+        with:
+          options: "--check --verbose --line-length 127"
+          src: "./scAR"
```

Comment:
Checks if the code has been formatted with black. Locally to format the code you need to run:
`black --line-length 127 scAR`